src/main/resources/templates/clientSide/clientSideList.html

```diff
@@ -0,0 +1,78 @@
+<head>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css" integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+    <link rel="stylesheet" th:href="@{/css/style.css}"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta charset="UTF-8">
+
+    <title>Šifry</title>
+
+</head>
+<body>
+
+<!--Navbar-->
+<nav class="navbar navbar-default navbarRed" >
+    <div class="navbarContentCentered">
+        <p class="bold floatLeft" th:text="${team.name}"></p>
+        <p class="floatLeft">Body:xx b</p>
+    </div>
+
+        <!-- Collapse button -->
+        <button class="btn" type="button" data-toggle="collapse" data-target="#navbarSupportedContent1"
+                aria-controls="navbarSupportedContent1" aria-expanded="false" aria-label="Toggle navigation"><span class="dark-blue-text">
+            <i class="fa fa-bars"></i></span></button>
+        <!-- Collapsible content -->
+        <div class="collapse navbar-collapse collapsedNavbar" id="navbarSupportedContent1">
+            <!-- Links -->
+            <ul class="navbar-nav mr-auto">
+                <li class="nav-item collapsedActive">
+                    <a class=" nav-link" href="#">Šifry <span class="sr-only">(current)</span></a>
+                </li>
+                <div class="dropdown-divider-cst"></div>
+                <li class="nav-item collapsedNotActive">
+                    <a class=" nav-link" href="#">Kartička</a>
+                </li>
+                <div class="dropdown-divider-cst"></div>
+                <li class="nav-item collapsedNotActive">
+                    <a class=" nav-link" href="#">Kontakty</a>
+                </li>
+                <div class="dropdown-divider-cst"></div>
+                <li class="nav-item collapsedNotActive">
+                    <a class=" nav-link" href="#">Odhlásit se</a>
+                </li>
+                <div class="dropdown-divider-cst"></div>
+            </ul>
+        </div>
+</nav>
+
+<div class="bodyContent">
+    <h2  class="botMargin10">Sifry</h2>
+    <div th:each="cypher : ${cyphers}" >
+            <div class="row">
+                <div th:text="|${cypher.stage}. ${cypher.name}|" class="col"></div>
+                <div th:unless="${#lists.isEmpty(cypher.hints)}" class="col hintColumn">
+                    <a href="#">
+                        <i class="far fa-lightbulb"></i>
+                    </a>
+                </div>
+                <div th:if="${status.name() == 'SKIPPED'}" class="col statusColumn">
```

Comment:
Radeji bych pouzil porovnavani primo enumu `th:if="${status == T(...package.Status).SKIPPED}"` a zaroven bych mozna pouzil switch. Mozna to pujde vymyslet i tak, aby tam byl element <i> jen jednou a jen se menila jeho class.